docs-chef-io/content/inspec/common_errors.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Chef InSpec Common Errors"
+draft = false
+gh_repo = "inspec"
+
+[menu]
+  [menu.inspec]
+    title = "Common Errors"
+    identifier = "inspec/reference/common_errors.md Chef InSpec Common Errors"
+    parent = "inspec/reference"
+    weight = 150
++++
+
+This section documents some of the Common Errors encountered while using Chef Inspec.
+
+* `Undefined local variable or method error for 'aws_eks_clusters' resource`
+
+    Syntax of using AWS resource in InSpec profile to test eks clusters:
+
+    ```bash
+        describe aws_eks_clusters.where( failed: true ) do
+            it { should_not exist }
+        end
+    ```
```

Comment:
Perhaps add a note that this error can occur with any of several AWS resources, not just aws_eks_clusters.

Reply:
Can this happen with any of the resource packs, just AWS, or is it specific AWS resources?

Reply:
@IanMadd This can happen with any AWS resource. I am gonna mention that here now.

Reply:
Couldn't I get the same error with GCP or Azure though if I didn't create a profile for those resource packs and then called one of their resources?

We should also have the instructions for creating a profile in docs.chef.io. There's instructions for GCP, but not the others for some reason. https://docs.chef.io/inspec/platforms/#create-an-inspec-profile-that-makes-use-of-inspec-gcp

Reply:
That makes sense. Let me think about someway I can include this fact for all the cloud resources because I was relying on an error to be mentioned on this page and then explain whats and whys around that.

And yes instructions page for other cloud resources is a good idea again. @clintoncwolfe should this documentation be covered by cloud resource pack team or could we also pick this up in future?